src/components/Modals/PublishCollectionModalOld/PublishCollectionModal.container.ts

```diff
@@ -0,0 +1,33 @@
+import { connect } from 'react-redux'
```

Comment:
Would you mind (if it doesn't exist) creating a ticket to remove this modal?

Reply:
https://github.com/decentraland/builder/issues/2044 The issue was already there but I added an extra comment for more context